addon/inline-components/read-only-content-section.js

```diff
@@ -0,0 +1,27 @@
+import { InlineComponentSpec } from '@lblod/ember-rdfa-editor/model/inline-components/model-inline-component';
+import { isElement } from '@lblod/ember-rdfa-editor/utils/dom-helpers';
+import { readOnlyContentSectionStatic } from './read-only-content-section-static';
+export default class ReadOnlyContentSectionSpec extends InlineComponentSpec {
+  matcher = {
+    tag: this.tag,
+    attributeBuilder: (node) => {
+      if (isElement(node)) {
+        if (
+          node.classList.contains('inline-component') &&
+          node.classList.contains(this.name)
```

Comment:
I think matching on the name is good, but better if we put it in a data-attribute
I think thats a better pattern in general for components to absolutely avoid any css conflicts

Reply:
Yes indeed, I am now refactoring inline components anyways, so I will move them to data attributes.